assets/scss/components/_modal.scss

```diff
@@ -103,6 +104,16 @@ lays out as a dialog on background content on larger screens.
 	z-index: map-get($zindex-map, "modal");
 }
 
+.view--modal {
+	.stripe-heroContent {
```

Comment:
There is, but it requires more work that doesn't belong in this PR. I filed a ticket for a separate cleanup task:
https://meetup.atlassian.net/browse/SDS-253